LEGO1/lego/legoomni/include/legorace.h

```diff
@@ -17,6 +17,26 @@ class RaceState : public LegoState {
 		inline MxS16 GetUnknown0x02() { return m_unk0x02; }
 		inline MxU16 GetScore() { return m_score; }
 
+		inline MxResult Serialize(LegoFile* p_legoFile)
+		{
+			if (p_legoFile->IsReadMode()) {
+				p_legoFile->Read(&m_id, 1);
+				p_legoFile->Read(&m_unk0x02, 2);
+				p_legoFile->Read(&m_score, 2);
+			}
+			else if (p_legoFile->IsWriteMode()) {
+				// A write variable needs to be used here, otherwise
```

Comment:
We don't really need this comment in every instance of this pattern. We should remove it everywhere